### PR TITLE
Fix/no wallet message

### PR DESCRIPTION
**Pull request by thefiend**
Added wallet provider selector and no wallet address error page

### CODE REVIEW

**Comment by Nebulis** [comment]
44 files change a fix ? =)

File: src/components/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node
+  ]).isRequired,
+  className: PropTypes.object,
+  onClick: PropTypes.func,
+  style: PropTypes.object,
+  rest: PropTypes.object,
+  primary: PropTypes.bool,
+  danger: PropTypes.bool,
```
Comment (sumitnitsurat) [inline src/components/Button/index.js]: pass these as a class.

File: static/colors.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const colors = () => ({
+  "orange-light": "#FBA54F",
+  "orange-dark": "#F5804D"
```
Comment (sumitnitsurat) [inline static/colors.js]: put colors in css variables and use them

**Comment by thefiend** [comment]
Edited as suggested. Please review again.

File: package.json [hunk 3]
```diff
@@ -13,22 +13,35 @@
...
+    "@zeit/next-sass": "^1.0.1",
+    "classnames": "^2.2.6",
     "debug": "^4.1.1",
     "eslint-plugin-react": "^7.7.0",
+    "ethers": "^4.0.27",
+    "jest-mock": "^24.8.0",
     "lodash": "^4.17.5",
     "next": "^8.0.0",
-    "next-redux-wrapper": "^1.3.5",
+    "next-ga": "^2.3.4",
```
Comment (rjchow) [inline package.json]: remove this if not used

File: src/components/Images.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
+/* eslint react/display-name: 0 */
+import React from "react";
+
+const Images = {
```
Comment (rjchow) [inline src/components/Images.js]: why not have svgs as svg files and import them properly?

https://github.com/twopluszero/next-images

File: src/config/index.js [hunk 5]
```diff
@@ -8,4 +8,6 @@ export const NETWORK_TYPES = {
   MOCK: "MOCK"
 };
 
+export const GA_ID = "";
```
Comment (rjchow) [inline src/config/index.js]: remove

**Comment by rjchow** [comment]
image
Not sure if I should get this message if I haven't logged in

**Comment by rjchow** [comment]
image
UI issues, also that font is really hard to read for the hash

**Pull request closed without merging**